ui-test/suite/task-test.ts

```diff
@@ -0,0 +1,75 @@
+/*-----------------------------------------------------------------------------------------------
+ *  Copyright (c) Red Hat, Inc. All rights reserved.
+ *  Licensed under the MIT License. See LICENSE file in the project root for license information.
+ *-----------------------------------------------------------------------------------------------*/
+import { ActivityBar, WebDriver, VSBrowser, Workbench, BottomBarPanel, EditorView } from 'vscode-extension-tester';
+import * as path from 'path';
+import { viewHasItems, terminalHasText, getNotificationWithMessage} from '../common/conditions';
+import { setInputTextAndConfirm, loginToOpenShiftWithTerminal, logoutFromOpenShiftWithTerminal, inputConfirm } from '../common/utils';
+import { commands, views } from '../common/constants';
+
+export function taskTest(clusterUrl: string, username: string, password: string): void {
+  describe('Task tests', () => {
+    let driver: WebDriver;
+    const simpleTaskFile = 'simple-task.yaml';
+    const taskName = 'echo-hello-world';
+
+    before(async function() {
+      this.timeout(200000);
+      driver = VSBrowser.instance.driver;
+      await new EditorView().closeAllEditors();
+      loginToOpenShiftWithTerminal(clusterUrl, username, password);
+
+      const control = new ActivityBar().getViewControl(views.TEKTON_TITLE);
+      await (await control).openView();
+      await (await new ActivityBar().getViewControl(views.TEKTON_TITLE)).openView();
```

Comment:
The first two lines together make the third line. I think you only need to open the view once, so I'd keep the third line only.